appmap/_implementation/configuration.py

```diff
@@ -55,147 +56,137 @@ def packages(self):
         return self._config['packages']
 
 
-def name_in_set(name, which):
-    return any(filter(lambda n: name.startswith(n + '.') or name == n,
-                      which))
-
-
-def class_in_set(class_name, which):
-    return name_in_set(class_name + '.', which)
-
-
-def function_in_set(fn, which):
-    class_name, fn_name = split_function_name(fn)
-    if class_name is None:
-        # fn isn't in a class, ignore it.
-        return False
-
-    ret = (class_in_set(class_name, which)
-           or name_in_set('%s.%s' %(class_name, fn_name), which))
-
-    logger.debug(('function_in_set, class_name %s'
-                  ' fnname %s'
-                  ' which %s'
-                  ' ret %s'),
-                 class_name,
-                 fn_name,
-                 which,
-                 ret)
-    return ret
+def splitname(obj):
```

Comment:
Seems like `splitname` could be useful in other contexts as well. Maybe it belongs in `utils.py`?

Reply:
Perhaps, although I thought it makes more sense to keep it near the client code until and unless there's another usage. `utils` packages are always a bit of annoying indirection when reading code (and I've read good arguments that they're bad style, though I'm not entirely convinced).

Reply:
Yeah, I'm not entirely convinced, either. I think it's ok for some things to wind up in a catch-all. I guess this can get moved the first time somebody wants to write `from .configuration import splitname`....

Reply:
> I guess this can get moved the first time somebody wants to write `from .configuration import splitname`....

Exactly my line of thought. YAGNI, but constant refactoring :)